File: src/main/java/io/openfuture/openmessanger/web/response/LastMessage.java
package io.openfuture.openmessanger.web.response;

import java.time.LocalDateTime;

import lombok.Value;

@Value
public class LastMessage {
    String chatUid;
    boolean group;
    String chatRoomName;
    Integer memberCount;
    String displayUserName;
    String lastMessageText;
    LocalDateTime lastMessageTime;
    String chatRoomPicture;
}

File: src/main/kotlin/io/openfuture/openmessenger/web/response/LastMessage.kt
package io.openfuture.openmessenger.web.response

import java.time.LocalDateTime

data class LastMessage(
    var chatUid: String? = null,
    var group: Boolean? = false,
    var chatRoomName: String? = null,
    var memberCount: Int? = null,
    var displayUserName: String? = null,
    var lastMessageText: String? = null,
    var lastMessageTime: LocalDateTime? = null,
    var chatRoomPicture: String? = null
)